eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,7 +1,21 @@
+resources:
+  repositories:
+    - repository: azure-sdk-tools
```

Comment:
After your updates to use eng/common is this needed for anything else? If not I'd suggest we remove it.